FreestarAds-Ogury.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |spec|
+  spec.name                = "FreestarAds-Ogury"
+  spec.version             = "1.0.0-beta-1"
```

Comment:
@carlosalcala-freestar Since this is your first adapter, this adapter should be version:
2.1.0.0-beta1
According to Ogury, I'm assuming we are using their version 2.1.0 (https://ogury-ltd.gitbook.io/release-notes/ios/ogury-sdk)  
For example, 2 months later we release a new version of ogury adapter and we are still using their 2.1.0, then:
2.1.0.1-beta1
Also, for example, if 6 months later ogury updates their SDK to 2.2.5 and we decide to update our adapter to it, then we would release 2.2.5.0
thank you!

Reply:
Fixed